code/lighting/lighting.cpp

```diff
@@ -148,6 +150,12 @@ static void light_rotate(light * l)
 	}
 }
 
+void light_add_directional(const vec3d* p0, float r1, float r2, const hdr_color* new_color)
+{
+	Assert(new_color!= nullptr);
+	light_add_directional(p0, r1, r2, new_color->i(), new_color->r(), new_color->g(), new_color->b());
```

Comment:
Several function calls don't match a visible prototype/definition here.

Reply:
Copy-paste oversights. I think I got them all in my last push